action.yaml

```diff
@@ -160,8 +168,10 @@ runs:
           local input_value="$2"
           local default_value="$3"
         
-          if [ ! -z "$input_value" ] && [ "$input_value" != "$default_value" ]; then
-            echo "$var_name=$input_value" >> $GITHUB_ENV
+          if [ ! -z "$input_value" ]; then
```

Comment:
Can you add comments here (as you wrote in https://github.com/aquasecurity/trivy-action/pull/454#issuecomment-2760919749)?